### DISCUSSION TITLE
Make robots suicide if they stay at spawn for more than 2 minutes

### DISCUSSION THREAD

**Discussion by leeroytm** [TitanTF/TitanTF#30] [General]
Suggestion by:  每個都你老婆 你有沒有愛滋阿#5120 <@482575549273997312>
Located: https://discord.com/channels/774245035615977473/845235964712583208/849128875367333948

---
Description: medbot not moving、bots not moving after bomb disappear(bomb reset)

**Comment by Lecubon1** [comment]
Probably for a issue with the AI myst using it
or so on my thought

**Comment by jiaweing** [comment]
This one is good
